Shoko.Server/API/v3/Models/Shoko/Episode.cs

```diff
@@ -192,7 +240,12 @@ public class TvDB
             /// <summary>
             /// Episode Title, in the language selected for TvDB. TvDB doesn't allow pulling more than one language at a time, so this isn't a list.
             /// </summary>
-            public string Title { get; set; }
+            public string PreferredTitle { get; set; }
```

Comment:
My comment in the doc xml says it all. PreferredTitle and Titles aren't relevant to TvDB. We kept the model for providers separate because the data that a user of the API should expect should be clear by the schema. We don't want to always give a list and a PreferredTitle if there's only ever going to be one title.

Reply:
Removed the interfaces, and reverted most of the cosmetic changes.